<?xml version="1.0"?>
<launch>
    <node name="simple_goal_publisher_drone1" pkg="simple_traj_airsim" type="simple_traj_airsim_node" output="screen">
        <param name="vehicle_name" value="Drone1"/>
    </node>

</launch>
